modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/utils/constants.ts

```diff
@@ -23,6 +23,12 @@ const FUZZY_OPTIONS = {
 	pre: '<strong>',
 };
 
+const HEADERS = {
+	'Accept': 'application/json',
+	'Accept-Language': Liferay.ThemeDisplay.getBCP47LanguageId(),
+	'Content-Type': 'application/json',
```

Comment:
Should we apply this constant to headers everywhere in DSM? There are 20 places where we set headers. Applying `Accept` and `Content-Type` seems like a nice way to reduce code duplication. Is it safe to apply `Accept-Language` everywhere?

Reply:
The idea is to include this `HEADERS` in every `GET` request, even if they are not related with the translation bug. However, to prevent causing conflicts with Kevin's PR I'd prefer to apply it as we remove/migrate each testcase.

Reply:
It seems that we cannot apply everywhere. I've just found that [this POST request to save fields doesn't work because it uses `multipart/form-data`](https://github.com/liferay-frontend/liferay-portal/pull/4332/files#diff-db30e74384e5bbddae379b185734ee38e904f718ec4863a16806ecfc51b4ce5cR499).

Any other request method seems to work fine with these headers. `DELETE` usually returns nothing and `PATCH` returns an updated response that needs the headers in order to work properly.